runtime/near-vm-runner/src/vm_kind.rs

```diff
@@ -1,8 +1,10 @@
 use borsh::BorshSerialize;
-use near_primitives::checked_feature;
 use near_vm_logic::ProtocolVersion;
 use std::hash::Hash;
 
+#[cfg(target_arch = "x86_64")]
+use near_primitives::checked_feature;
```

Comment:
Now that you changed cfg on `pub fn for_protocol_version`, this include is actually needed again^^ (Compilation on macbook fails again)

It looks like it's not needed, since we only use `checked_feature` inside `cfg!(target_arch = "x86_64") { ... }` but I believe because the way macros are evaluated, we need the include here.

Reply:
Oh right. Fixed now, thanks! (You're right in that the `cfg!` macro just evaluates to true or false and doesn't prevent compilation of what's inside, as opposed to the `#[cfg()]` attribute that does prevent compilation)